liblepton/scheme/netlist/schematic-connection.scm

```diff
@@ -100,8 +101,12 @@ Example usage:
                  (_ #\?)))
              args)))))
 
+;;; This is a helper object property to get object connections
+;;; once.
+(define object-conns (make-object-property))
```

Comment:
How does this speed up the test suite?
BTW, now `make distcheck` runs for 6 minutes
on my machine instead of 5.

Reply:
Well, without this property, the code may work twice as long.  It is especially apparent with the `stack-torture.sch`.  Reset the branch to the state where the function `object-connections` has been actually rewritten, and you'll see what I'm talking about.